### PR TITLE
feture(#056): Added index page

### PR DESCRIPTION
**Pull request by IsmaelP19**
The index page has been added. The redirections have been changed all to /catalogue when it has to redirect to the catalogue. The / direction is reserved for the index page.

To my reviewers, please check that all redirections are right, I may have left some of them.

### CODE REVIEW

File: static/styles/index.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  text-decoration: none;
+  padding: 10px;
+  border: 2px solid #ffca2c;
+  background-color: #ffca2c;
+  border-radius: 40px;
+  color: #000;
+  font-size: 1.5rem;
+  font-family: 'Courier New', Courier, monospace;
+  font-weight: bold;
+  padding: 10px 20px;
```
Comment (migromarj) [inline static/styles/index.css:59]: Padding should have only a value.

**Review by migromarj — CHANGES_REQUESTED** [review]
The page works fine and the layout is amazing, but you should fix the 'padding' that has two values ​​assigned.

File: static/styles/index.css (hunk 1 above)
Comment (IsmaelP19) [reply static/styles/index.css:59]: You are right!! I'm changing it now ⏱️

**Review by ezegonmac — APPROVED** [review]
I think all redirects works fine. Great job 👍🏻

**Review by migromarj — APPROVED** [review]
Thank you for solving what I mentioned in the review. Incredible work, the page has been spectacular 😮

**Pull request merged**